src/app/components/content-finder-page/content-finder-page.component.ts

```diff
@@ -33,7 +33,7 @@ export class ContentFinderPageComponent implements OnInit {
    /**
     * Sets defualt for content selection to All
     */
-   selFormat: string = "All";
+   selFormat: string[] = ["Code", "Document", "Powerpoint"];
```

Comment:
This is good, although with these changes, the functionality to search for flagged content is gone. Was this functionality removed in favor of the new search mechanic? If so, that's fine. Just want to make sure that no previous features are lost unintentionally.

Reply:
Flagged never worked in the first place unfortunately.  The ability to see flagged content is viable in my version however.  You simply do not specify any modules to restrict the search by and flagged content comes up along with everything else.  My recommendation in the future however is that the Module search bar has an option for retrieving only Flagged content.  The reason is that Flagged refers to content without a Module and has absolutely nothing to do with the Format of the content.

Reply:
Alright, that's fine. Make sure to mention this in showcase for future iterations then.